pipeline-scripts/release.groovy

```diff
@@ -452,6 +456,45 @@ def void sendReleaseCompleteMessage(Map release, int advisoryNumber, String advi
     }
 }
 
+def createAdvisoriesFor(ocpVersion) {
+    build(
+        job: "/build/advisories",
+        propagate: false,
+        parameters: [
+            string(name: "VERSION":       value: ocpVersion),
+            string(name: "DATE",          value: determineNextReleaseDate(ocpVersion)),
+            string(name: "ASSIGNED_TO",   value: "openshift-qe-errata@redhat.com"),
+            string(name: "MANAGER",       value: "vlaad@redhat.com"),
+            string(name: "PACKAGE_OWNER", value: "lmeyer@redhat.com"),
```

Comment:
could most of these be left as the defaults? hate to hardcode our names any more than necessary :)

Reply:
for sure! done here: https://github.com/openshift/aos-cd-jobs/pull/2307/commits/47c3e981a355081b33c5a81ddfdcf5b14df047b5